### ISSUE TITLE
Normalization of tiff files

### ISSUE THREAD

**Issue by JohanRaniseth** [junyanz/pytorch-CycleGAN-and-pix2pix#950]
Hey I'm working with a dataset of tiff files where the images aren't confined to values in the range [0,255] but instead have values upwards of 1400. I've understood that I need to normalize my data so would my best solution be to remove the get_transform line [here](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/blob/master/data/aligned_dataset.py#L50) which I assume does the normalization, and instead normalize all my images through a python script? And then use my normalized images as my training data?

Also a side question, when normalizing f.ex. jpgs all images have the same min/max values from the range [0,255] but since tiff min/max values vary, should I find the min/max values of my whole tif dataset and use these for my normalization? It feels wrong to scale two images from the range [1000,1400] to [-1,1] and [1100,1300] to [-1,1] instead of something like [1000,1400] to [-1,1] and [1100,1300] to [-0.5,0.5].
Any help would be greatly appreciated.

Best,
Johan

**Comment by junyanz** [comment]
Yes, it's correct. You need to find the min and max of the whole dataset. You can modify this [line](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/blob/master/data/base_dataset.py#L106).

**Comment by JohanRaniseth** [comment]
Thank you for confirming!
